distribution/std-lib/Base/src/Main.enso

```diff
@@ -95,3 +97,4 @@ Any.== that = if Meta.is_same_object this that then True else
            Therefore, there is no more cases to handle in this method.
         _ -> False
 
+Boolean.to_json = Json.Boolean this
```

Comment:
This should probably go in a `Data.Boolean.Extensions` module like we have for `Number` and `Text`.